src/libs/brands.ts

```diff
@@ -151,6 +155,10 @@ export const brands: Record<string, BrandConfiguration> = {
         light: '#e9ecef',
       },
     },
+    datasetBuilderSnapshotId:
+      getConfig().terraDeploymentEnv === 'dev'
+        ? 'c3eb4708-444f-4cbf-a32c-0d3bb93d4819'
```

Comment:
do we want to use the gcp snapshot ids? since axin will be on gcp and it works better. I know the prod one doesn;t exist yet, but at least for dev maybe

Reply:
I know this is a good first step, but once this product is in use, we won't want to have to update this semi-regularly. I think we would want to move towards getting the snapshot id on the library page. maybe in refinement we could talk about the steps to follow up on this.

Reply:
I'll make a ticket for the backlog to track this. I don't think it's something we need to worry about until we have  a data release in production, or maybe if we get another dataset use case, so I would put off the work for now.

Reply:
I created https://broadworkbench.atlassian.net/browse/DC-1188for this issue